ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -353,6 +353,11 @@ final class Metrics(val registry: MetricRegistry) {
       val uploadPackages: Timer = registry.timer(Prefix :+ "upload_packages")
       val publishConfiguration: Timer = registry.timer(Prefix :+ "publish_configuration")
 
+      val decodeTransactionLogUpdate: Timer =
+        registry.timer(Prefix :+ "decode_transaction_log_update")
```

Comment:
Consider `transaction_log_update_decode` to improve the logical grouping when lexicographically sorting metric names. .